CMakeLists.txt

```diff
@@ -65,6 +65,12 @@ if(HIDAPI_WITH_TESTS)
     enable_testing()
 endif()
 
+if(WIN32)
+    option(HIDAPI_BUILD_PP_DATA_DUMP "Build small Windows console application pp_data_dump.exe" ${IS_DEBUG_BUILD})
+else()
+    set(HIDAPI_BUILD_PP_DATA_DUMP OFF)
```

Comment:
no CMake code outside `windows/CMakeLists.txt` would use it, so the entire wariable outside `if(WIN32)` is needed
```suggestion
```